2-1-2.py

```diff
@@ -0,0 +1,11 @@
+weight = float(input())
+growth = float(input())
+
+BMI = weight / (growth ** 2)
+
+if 18.5 <= BMI <= 25:
+    print('Оптимальная масса')
+elif BMI < 18.5:
+    print('Недостаточная масса')
+elif BMI > 25:
+    print('Избыточная масса')
```

Comment:
You can simply a little this if block:

```suggestion
if BMI < 18.5:
    print('Недостаточная масса')
elif BMI <= 25:
    print('Оптимальная масса')
else:
    print('Избыточная масса')
```